src/frontend/src/components/DownloadMetadataTemplate/common/constants.ts

```diff
@@ -14,22 +16,203 @@ export const BASE_METADATA_HEADERS = {
+      DATE_FORMAT, // collectionDate -- not valid, here as explainer in template
+      "North America/USA/California/Los Angeles County", // collectionLocation
+      DATE_FORMAT, // sequencingDate -- not valid, here as explainer in template
+      BOOLEAN_FORMAT, // keepPrivate -- not valid, here as explainer in template
+    ],
+    // Subsequent example rows are mostly valid, honest-to-goodness examples...
+    // ... except for the dates. This is to avoid Excel auto "correct".
+    [
+      EXAMPLE_SAMPLE_IDS[1], // sampleId
+      "id101", // privateId
+      "", // publicId -- optional, showing that with blank use
+      DATE_FORMAT, // collectionDate
+      "San Francisco County", // collectionLocation
+      "", // sequencingDate -- optional, showing that with blank use
+      "No", // keepPrivate
+    ],
+    [
+      EXAMPLE_SAMPLE_IDS[2], // sampleId
+      "id102", // privateId
+      "USA/CA-CZB-0001/2021", // publicId
+      DATE_FORMAT, // collectionDate
+      "North America/USA/California/San Francisco County", // collectionLocation
+      DATE_FORMAT, // sequencingDate
+      "No", // keepPrivate
+    ],
+  ],
+  [Pathogen.MONKEY_POX]: GENERAL_VIRAL_UPLOAD_EXAMPLE_ROWS,
+};
+
+export function getEditExampleRows(
+  pathogen: Pathogen,
+  collectionLocation?: GisaidLocation
+): string[][] {
+  // return example rows with the countys collectionLocation
+  const exampleCollectionLocation = collectionLocation
+    ? `${collectionLocation.region}/${collectionLocation.country}/${collectionLocation.division}/${collectionLocation.location}`
+    : "North America/USA/Californa/Humboldt County";
+
+  const editRows: PathogenConfigType<string[][]> = {
+    [Pathogen.COVID]: [
+      [
+        EXAMPLE_CURRENT_PRIVATE_IDENTIFIERS[0], // currentPrivateID
+        "X3421876", // newPrivateID
+        "hCoV-19/USA/demo-17806/2021", // publicId
+        DATE_FORMAT, // collectionDate,
+        exampleCollectionLocation, //collectionLocation
+        DATE_FORMAT, // sequencingDate
+        BOOLEAN_FORMAT, // keepPrivate
+      ],
+      [
+        EXAMPLE_CURRENT_PRIVATE_IDENTIFIERS[1],
+        "SOP292344X", // newPrivateID
+        "hCoV-19/USA/demo-17807/2021", // publicId
+        DATE_FORMAT, // collectionDate,
+        exampleCollectionLocation, //collectionLocation
+        DATE_FORMAT, // sequencingDate
+        BOOLEAN_FORMAT, // keepPrivate
+      ],
+      [
+        EXAMPLE_CURRENT_PRIVATE_IDENTIFIERS[2],
+        "T2348ACT", // newPrivateID
+        "hCoV-19/USA/demo-17808/2021", // publicId
+        DATE_FORMAT, // collectionDate,
+        exampleCollectionLocation, //collectionLocation
+        "Delete", // sequencingDate
+        BOOLEAN_FORMAT, // keepPrivate
+      ],
+    ],
+    [Pathogen.MONKEY_POX]: [
+      [
+        EXAMPLE_CURRENT_PRIVATE_IDENTIFIERS[0], // currentPrivateID
+        "X3421876", // newPrivateID
+        "hMpxV/USA/demo-17806/2021", // publicId
```

Comment:
I'm not particularly concerned right now, but if we start to have to do this for more pathogens, we'll want to change this into a function that generates the array. Looks like the only thing I see changing is the pathogen prefix in the publicId (`hCoV-19` vs `hMpxV`).

Reply:
I agree.  The Upload example data has the GISAID -> GenBank change and the Edit example data has the publicId change.  All of the other data is the same.  I don't know off the top of my head an eloquent way to mix and match custom vs non-custom fields.  Using the generator idea from above might simplify this a bit.  Given the tight timeline of getting this project done and the uncertain outlook going forward, I'm leaning toward leaving this as is.  Maybe we will have some more time to revisit when we implement hepC.

Reply:
I agree, seems like the best move!